pbs/its
# OCTO-9164 convert images with icc profiles to srgb

This may be a smarter way to handle ICC profiles (alternative is https://github.com/pbs/its/pull/171)

Instead of passing through ICC profiles and hoping that clients know how to handle them, we can dynamically convert the images to the sRGB colorspace. Two reasons we should do this:

* browser support for ICC profiles is actually not bad these days (based on limited ad-hoc testing), but we probably can't assume good ICC profile support for OTT devices, etc.
* ICC profiles can add a surprising amount of weight to images - fine for printers, etc, but bad for serving images over the web.

Experimentally, images that look greyed-out in the current ITS look fine with this conversion logic applied (which is what the test in this PR attempts to check).

# To test locally:

* start the app: `docker-compose up`
* open http://localhost:5000/tests/images/jpeg_with_icc_profile.jpg

The image should not be greyed out. If you inspect this image, it should not have an ICC profile. (I'm not sure how to manually verify the latter - I assume something like photoshop or GIMP can tell you - we're checking it programmatically with the test change in this PR)

If you repeat the above steps on the master branch, you should see that the image looks fine (at least in modern browsers), but inspection will reveal that it still has an ICC profile attached.
